tests/unit/test_region_configuration.py

```diff
@@ -51,6 +52,16 @@ def test_when_apigw_edge_configuration_is_not_supported(self, partition):
     def test_is_service_supported_positive(self, service, region):
         self.assertTrue(RegionConfiguration.is_service_supported(service, region))
 
+    @parameterized.expand(
+        [
+            ["ec2", None],
+        ]
+    )
+    def test_is_service_supported_positive_boto3_default_session(self, service, region):
```

Comment:
It seems that `region`  isn't used at all.

Reply:
Good catch! let me remove that.